cmd/tarocli/addrs.go

```diff
@@ -141,7 +141,7 @@ func queryAddr(ctx *cli.Context) error {
 		start = startTime.Unix()
 	}
 
-	end := math.MaxInt64
+	end := int64(math.MaxInt64) //wrap so MaxInt64 will cross-compile on 32 bit arch
```

Comment:
Can you put the comment above the code and use proper spacing and punctuation please?